Shared/Samples/Identify layer features/IdentifyLayerFeaturesView.swift

```diff
@@ -0,0 +1,150 @@
+                            try await featureTable.load()
+                            let featureLayer = FeatureLayer(featureTable: featureTable)
+                            map.addOperationalLayer(featureLayer)
+                        } catch {
+                            // Present error load the layers if any.
+                            self.error = error
+                        }
+                    }
+                    .task(id: tapScreenPoint) {
+                        // Identify layers using the screen point.
+                        if let screenPoint = tapScreenPoint,
+                           let results = try? await proxy.identifyLayers(
+                            screenPoint: screenPoint,
+                            tolerance: 12,
+                            returnPopupsOnly: false,
+                            maximumResultsPerLayer: 10
+                           ) {
+                            handleIdentifyResults(results)
+                        }
+                    }
+                    .overlay(alignment: .top) {
+                        Text(overlayText)
+                            .frame(maxWidth: .infinity, alignment: .center)
+                            .padding(8)
+                            .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+                    }
+                    .alert(isPresented: $isShowingAlert, presentingError: error)
+            }
+        }
+    }
+}
+
+private extension IdentifyLayerFeaturesView {
+    /// Update the overlay text based on the identify layer results.
+    /// - Parameter results: An `IdentifyLayerResult Array` to handle.
+    func handleIdentifyResults(_ results: [IdentifyLayerResult]) {
+        // Create overlay message from the results.
+        var alertMessageString = ""
+        var totalGeoElementsCount = 0
+        for (i, identifyLayerResult) in results.enumerated() {
+            // Create alert message the from geoElement count and the layer name.
+            let geoElementsCount = geoElementsCountFromResult(identifyLayerResult)
+            let layerName = identifyLayerResult.layerContent.name
+            alertMessageString.append("\(layerName): \(geoElementsCount)")
+            
+            // Add new line character if not the final element in array.
+            if i < results.count - 1 {
+                alertMessageString.append("\n")
+            }
+            
+            // Update total count.
+            totalGeoElementsCount += geoElementsCount
+        }
+        
+        // Update overlay text with the geo-elements found if any.
+        if totalGeoElementsCount > 0 {
+            overlayText = alertMessageString
+        } else {
+            overlayText = "No element found."
+        }
+    }
+    
+    /// Count the geo-elements from an identify layer result using recursion.
+    /// - Parameter result: The `IdentifyLayerResult` to count.
+    /// - Returns: An `Int` count of the geo-elements.
```

Comment:
```suggestion
    /// - Returns: The count of the geo-elements.
```
I think the word "count" implicitly infers it is an integer.